apisupport.wizards/src/org/netbeans/modules/apisupport/project/ui/wizard/action/actionListener.javx

```diff
@@ -1,41 +1,22 @@
-<#--
-DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+<!--
```

Comment:
Template files for module development are not user visible at the moment.
https://github.com/apache/incubator-netbeans/blob/c3fcc0a3fdd8e6c7ab2055118798c77e06bae972/apisupport.wizards/src/org/netbeans/modules/apisupport/project/ui/wizard/common/layer.xml#L29-L92

So, I think that there is no problem to add the license header to them.
However, would you please fix the comment tag from `<!--` to `<#--`  if license headers for template file are not removed(template files are not added to rat excludes)?

Reply:
OK, didn't know there's a difference between  `<!--` and `<#--`

Can you tell me the difference?

Reply:
`<#-- -->` is a freemarker comment: http://freemarker.org/docs/dgui_template_overallstructure.html
`<!-- -->` is a XML/HTML comment.

This difference is important. Templates are processed by freemarker. So if `<#-- -->` is used in a template file, generated code does not have the comment block(`<#-- -->`). However, `<!-- -->` is used in it, generated code has the comment block(`<!-- -->`). In this case, the generated code will have the license header.

BTW, if template files are user visible(e.g. Tools > Templates > Select Something > Open in editor), probably, we should not add the license header. (we should add them to rat excludes.)
https://lists.apache.org/thread.html/43bec9e29901ef8d935c2f1515c2b7710c6a7e0b32657c3ddc1468c4@%3Cdev.netbeans.apache.org%3E

Reply:
These licenses are not user visible, so I will change <!-- back to <#--

Thanks.

Reply:
Done as described above, in https://github.com/apache/incubator-netbeans/pull/145/commits/60e67c476790fa10c282797fe9715d80e2405469.